### ISSUE TITLE
ImageData should take the image in the constructor

### ISSUE THREAD

**Issue by fzimmermann89** [ckolbPTB/InhomCorr#16]
https://github.com/ckolbPTB/InhomCorr/blob/90c0b7af9546fd91e34a3b4bb13b55208a07bc4b/src/inhomcorr/interfaces/mr_data_interface.py#L67

I believe it would be usefuull to just be able to set the image in the constructor. There are lots of places were we otherwise have the sequence of 
```
img=ImageData()
img.data=image
```
I would even go so far to have the image mandatory in the __init__. Soi we never have an imageData that does not have an image tensor.

**Comment by ckolbPTB** [comment]
I agree or can anyone think of a use case where an empty ImageData object would be required?

**Comment by schuenke** [comment]
I also vote for including the image in the constructor. I cannot think of any case where we need an empty object. However, having the option to change the image data later makes sense imo.
